llama.cpp

```diff
@@ -68,6 +68,7 @@
 #include <cstdio>
 #include <cstring>
 #include <ctime>
+#include <cwctype>
```

Comment:
Should also `#include <locale>` for std::locale and the locale-aware version of tolower.

Reply:
Yup.